src/ad/adReducer.js

```diff
@@ -288,17 +298,30 @@ export default function adReducer(state = initialState, action) {
     }
 }
 
-function* getAd(action) {
+function* getRekrutteringsbistandstilling(action) {
```

Comment:
Husker ikke hva vi konkluderte med da vi snakket om bounded contexts. Var det greit å bare kalle dette for en _stilling_ i denne appen?

Reply:
Stilling finnes allerede. Og er også en parameter inne i dette objektet... Så vi har en unbounded context om det er noe som heter det. Der den ene contexten inneholder den andre. I steden for at de er ved siden av hverandre. Vi kunne behandlet det som to ting, men har valgt å ha ett objekt i ett endepunkt... Kanskje vi egentlig kunne sendt det over som to dtoer, men vet ikke om det ville blitt bedre.